src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -205,3 +206,19 @@ export const hasMonthYear = (err, fieldData) => {
     err.addError('Please provide both month and year');
   }
 };
+
+export const isWithinServicePeriod = (errors, fieldData, formData) => {
+  const inServicePeriod = _.get(
+    'serviceInformation.servicePeriods',
+    formData,
+    [],
+  ).some(pos => isWithinRange(fieldData, pos.dateRange));
+  if (!inServicePeriod) {
+    const dateIsComplete = dateString =>
+      dateString && !dateString.includes('X');
+    if (dateIsComplete(fieldData.from) && dateIsComplete(fieldData.to)) {
```

Comment:
We don't allow partial entry for POW, right? So this basically waits until the user is finished inputting the dates before the sirens start up?

Reply:
Right, yeah. I think they have to enter the whole date.

That's exactly it.